src/web.ts

```diff
@@ -14,6 +15,14 @@ export function createWebTracker(client: Client, prefix?: string): WebTracker {
 			client.track(`${prefix ? `[${prefix}] ` : ''}${name}`, props);
 		},
 
+		trackNavigationClick(props?: Properties) {
+			this.track('Navigation Click', {
+				...props,
+				current_url: window.location.href,
+				current_url_path: window.location.pathname,
```

Comment:
@gelbal  If its a Navigation Click, shouldnt we add also the <a href> value too just in case? ie: The [Hub] Project Navigation click already has it, and it could be useful to ask for it on all Navigation Clicks I believe.

Reply:
You mean the `href` value, right? Yeah, I wanted to capture it automatically but I don't know if there is a clean & straightforward way to capture it on `analytics-client` (like we capture the `window.location` values). Is there?

Reply:
Thinking it further now, if we are not able to capture `href` automatically here, it'd be better to require the `href` property in this function. So the component maintainers know that they have to pass the destination of the navigation.

Reply:
Lol I just edited my comment because since I wrote the href tag explicitly it got itnerpreted as link haha. Yes, I mean the href of any button or "<a>" element. 

And I cant think of a way to get that since its to tied up to the front, but 
> t'd be better to require the href property in this function

this sounds perfect for this case.

Reply:
Cool. I made that change now to require `href` parameter.

Reply:
Looks like a nice quality of life improvement. Would we like to mention it in the README as well?

Reply:
Sure, why not. I edited Readme. Please take another look Pranas.